pkg/ytconfig/canondata/TestGetExecNodeConfigWithCri/single-container-without-job-resources/test.canondata

```diff
@@ -127,6 +127,9 @@
                     namespace=yt;
                     "base_cgroup"="/yt";
                 };
+                "cri_image_cache"={
+                    capacity=4294967296;
+                };
```

Comment:
Why canondata config is changed if tests are not? Is it some default value (I don't see value in diff)?
Is it expected that config would change if no new fileds in spec is specified?

Reply:
cri image cache section is added for cri env setip

Reply:
I've added 4gb quota for image-cache location in previous pr

Reply:
i see, so if nothing new (cache quota) would be specified in real spec, we won't have a diff in configs?

Reply:
without explicit quota there will be volume storage request.